### ISSUE TITLE
Zwave communication not robust

### ISSUE THREAD

**Issue by JoopBanaan** [athombv/homey#2334]
**Metadata**

* **Homey Version**:1.5.8
* **Homey Log Code**:
* **Smartphone Platform**:
* **Smartphone App Version**:

**If related to a Homey App:**
* **Homey App Version**: Fibaro 1.5.17
* **Homey App Log Code**:
* **Affected Devices (preferably product codes)**: Fibaro FGD-212, dimmer 2, firmware V3.5

**What did you try to achieve?**

Turn multiple Fibaro dimmers on at the same moment 

**What did you expect as result?**

That every time, all the dimmers turn on

**What was the result?**

Most of the times, not all dimmers turn on

**Any other remarks**
If I schedule the turning on by delaying each command so that only 1 command is sent out every 2 seconds, reliability is much better: almost all the times all the dimmers turn on.

Theory: I have a large Zwave network ~40nodes, with most devices that are reached by multiple hubs over the mesh network. Scheduling of the Zwave commands is not done fully robust, so together with the delays impaired with the multiple hubs, results in non-robust communication

**Comment by JoopBanaan** [comment]
Just tested with homey 1.5.11 firmware: turning 4 z-wave modules off at same moment: only 1 module turns off , rest remains on. => Not solved by 1.5.11

**Comment by bvdbos** [comment]
Turning multiple devices on/off is something every controller will have issues with (like you  already imply with the "multiple hubs". Fortunately in zwave one can use association-groups:
https://manuals.fibaro.com/knowledge-base-browse/associations/

**Comment by JoopBanaan** [comment]
Yes one can use in some cases association. When you want to turn all lights off in a (large) house, it does not work, since associations do not work when hubs are in between.

The fact that multiple hubs are in between should not stand in the way to get a robust solution. With proper scheduling a robust solution can still be obtained, IMHO

Do we know of other controllers that can turn off multiple devices at once in a robust way? If so, how do they accomplish that?

**Comment by ChristerJ83** [comment]
This problem is also present for a smaller set of devices. I have only 8 devices (7 of them are z-wave, one Wifi) in a flow and still it is more or less 50/50 if all of the devices are turned on/off every time the flow runs.
The Wifi device never fails btw.
The flow is also configured with a delay so that there is 5 sec between every device is supposed to be activated - it does not help.
Non of the devices are battery devices.
